index.xml

```diff
@@ -175,6 +175,23 @@ of keys and assignment of `keyId` is out of scope for this document.
      </t>
     </section>
 
+    <section anchor="date" title="date">
+     <t>
+REQUIRED.  The `date` field is an integer representing a Unix time. Represent the
```

Comment:
about MUST/SHOULD: let's discuss that *after* we found a suitable spec. A good solution could be to *version* the Signature header so that we can fix future security issues without breaking old stuff.